EditorExtensions/Settings/LegacySettings.cs

```diff
@@ -80,6 +80,7 @@ public void ApplyTo(WESettings target)
             target.JavaScript.GenerateSourceMaps = GetBoolean("JavaScriptGenerateSourceMaps");
             target.JavaScript.GzipMinifiedFiles = GetBoolean("JavaScriptEnableGzipping");
             target.JavaScript.BlockCommentCompletion = GetBoolean("JavaScriptCommentCompletion");
+			target.JavaScript.BaseUrl = GetString("JavaScriptBaseUrl");
```

Comment:
This is not required; legacy settings are obsolete. This importer is for scenarios if someone opens an year old project with old settings files in it.